Sources/Helpers/UIFont+DynamicTypeCustom.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  UIFont+DynamicTypeCustom.swift
+//  Pods
+//
+//  Created by Nikki Vergracht on 22/05/2017.
+//
+//
+
+import UIKit
+
+/// Available from iOS & tvOS 7.0 upwards.
+@available(iOS 7.0, tvOS 7.0, *)
+extension UIFont {
+    
+    /// Uses the `fontDescriptor` and `fontFamily` provided to return a custom UIFont compatible with the Dynamic Type selected in the user's settings.
+    private class func preferredFont(forFontDescriptor fontDescriptor: UIFontDescriptor, withFontFamily fontFamily: String) -> UIFont {
```

Comment:
I followed Apple in the function naming process. For now I have updated all the classes to be more Swift like and added some documentation.

The following method is one provided by Apple: `open class func preferredFont(forTextStyle style: UIFontTextStyle, compatibleWith traitCollection: UITraitCollection?) -> UIFont`. Please review if the change to a more Swift like naming process is ok after looking at Apple's example.